src/store/index.js

```diff
@@ -13,6 +13,7 @@ export const initialState = () => ({
 		triggers: [],
 		departments: [],
 		resources: {},
+		businessUnitIds: [],
```

Comment:
I feel like this is not the right place to store this information. All properties here are supposed to come from the server and this is not gonna happen with `Business Unit`. This new feature is client-side, so my suggestion is to store the `Business Unit` inside the `Iframe` property and read it from there.

BTW: What happens when `clearLocalStorageWhenChatEnded` is called? The `Business Unit` should not be affected.

Reply:
Yeah, that makes sense. I've moved it to the root level instead now. I couldn't store it on Iframe prop since that is getting reset when `clearLocalStorageWhenChatEnded` is set. Keeping it on a root level now prevents it from getting cleared